landscape/client/package/reporter.py

```diff
@@ -664,6 +670,13 @@ def _compute_packages_changes(self):
                 if self._facade.is_package_upgrade(package):
                     current_upgrades.add(id)
 
+                # Is this package present in the security pocket?
+                security_origins = [
```

Comment:
given you're treating this as a boolean, you could escape early by using `any()`